Training_analysis.ipynb

```diff
@@ -3152,7 +3183,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.12"
+   "version": "3.10.6"
```

Comment:
Our docker containers are currently running Python 3.7; so things should be edited / saved as such.

Reply:
Reverted